### DISCUSSION TITLE
Difference between `sync` and `update`?

### DISCUSSION THREAD

**Discussion by kflak** [lumen-oss/rocks.nvim#326] [Q&A] [answered]
Found myself getting a bit confused about the difference between the two... Could someone please explain?

**Comment by NTBBloodbath** [comment, marked as answer]
Hey there!

As said in the usage section of the README, `update` will attempt to update every available rock if it's not pinned to a certain version, while `sync` will synchronize the current Neovim configuration with the `rocks.toml` contents.

That means, running `:Rocks sync` will attempt to install missing rocks, uninstall unneeded rocks and make sure every installed rock has the correct version as our `rocks.toml` file also works as a "lockfile" on itself.

**Comment by kflak** [comment]
Ok, that makes things clearer for me. Thanks a lot!
